docs/workflows/doi.md

```diff
@@ -1,4 +1,120 @@
+```
+
+Once the publications table has been created, the publications are grouped by entity type and publication year. For instance publications are grouped by institution and publication year. The last step involves creating aggregate timeseries statistics based on the yearly groups of publications.
+
+``` eval_rst
+.. image:: ../static/diagrams/aggregate_publications.png
+   :width: 650
+```
+
+### Intermediate Datasets
+
+Digging into the mechanics of the workflow, there are a number of intermediate tables that are created. Much of the time, these are not relevant as the information is better organised in the final output tables. However, for specific analysis they may be useful so are included here for completeness 
+
+```eval_rst
++----------------+----------------------------------------------------+-----+-----------------------------------+
+| Name           | Table                                              | PID | SQL File Name                     |
++================+====================================================+=====+===================================+
+| Crossref Events| `observatory_intermediate.crossref_eventsYYYYMMDD` | DOI | create_crossref_events.sql.jinja2 |
++----------------+----------------------------------------------------+-----+-----------------------------------+
+| Crosref Fundref| `observatory_intermediate.crossref_fundrefYYYYMMDD`| DOI | create_crossref_fundref.sql.jinja2|
++----------------+----------------------------------------------------+-----+-----------------------------------+
+| Grid           | `observatory_intermediate.gridYYYYMMDD`            | GRID| create_grid.sql.jinja2            |
++----------------+----------------------------------------------------+-----+-----------------------------------+
+| MAG            | `observatory_intermediate.magYYYYMMDD`             | DOI | create_mag.sql.jinja2             |
++----------------+----------------------------------------------------+-----+-----------------------------------+
+| Open Citations | `observatory_intermediate.open_citationsYYYYMMDD`  | DOI | create_open_citations.sql.jinja2  |
++----------------+----------------------------------------------------+-----+-----------------------------------+
+| ORCID          | `observatory_intermediate.orcidYYYYMMDD`           | DOI | create_orcid.sql.jinja2           |
++----------------+----------------------------------------------------+-----+-----------------------------------+
+| Unpaywall      | `observatory_intermediate.unpaywallYYYYMMDD`       | DOI | create_unpaywall.sql.jinja2       |
++----------------+----------------------------------------------------+-----+-----------------------------------+
+
+```
+
+### Output datasets
+
+The final set of output dataset are as followed. Aside from the DOI table, which is an integrated set of all the input datasets linked by DOI, they each share a very similar, or identical, schema. The difference is around the entity being aggregated against.
```

Comment:
'The final set of output dataset are as followed' change to 'The final set of output datasets are as follows'.

Schema would be very useful and is requested- if the schemas are identical, use a common table and add the fields that are specific to each table.